### PR TITLE
fix(android): prevent removal of polyfills between builds (8_0_X)

### PR DESCRIPTION
**Pull request by janvennemann**
JIRA: https://jira.appcelerator.org/browse/TIMOB-26716

8_0_X backport of #10622

### CODE REVIEW

File: android/cli/commands/_build.js [hunk 1]
```diff
@@ -2796,6 +2796,38 @@ AndroidBuilder.prototype.copyResources = function copyResources(next) {
...
+						packageJsonPath = require.resolve(path.join(moduleId, 'package.json'));
+						packageJsonPath = path.join(nodeModulesPath, packageJsonPath.substring(packageJsonPath.indexOf('node_modules') + 12));
+					}
+					const modulePath = path.dirname(packageJsonPath);
+					Object.keys(this.lastBuildFiles).forEach(p => {
+						if (p.startsWith(modulePath)) {
+							delete this.lastBuildFiles[p];
+						}
+					});
+					const packageJson = fs.readJSONSync(packageJsonPath);
```
Comment (lokeshchdhry) [inline android/cli/commands/_build.js]: Build is failing at this line. 
`fs` does not seem to have `readJSONSync`, `fs-extra` does.

```
2019-01-16T22:04:03.137Z | ERROR  | An uncaught exception was thrown!
fs.readJSONSync is not a function
2019-01-16T22:04:03.138Z | ERROR  | fs.readJSONSync is not a function
```

**Review by sgtcoolguy — APPROVED** [review]
LGTM (it's ugly that we need to basically duplicate logic here, though)

**Comment by janvennemann** [comment]
Yes, i don't really like it either, but given the way how the Android build works, i don't see a more elegant solution without changing a lot more code.

**Comment by lokeshchdhry** [comment]
FR Passed.

No `java.io.FileNotFoundException: Resources/node_modules/core-js/modules/web.dom.iterable.js` error seen.

Studio Ver:     5.1.2.201812191831
SDK Ver:        8.0.0 local build
OS Ver:         10.14
Xcode Ver:      Xcode 10.1
Appc NPM:       4.2.13
Appc CLI:       7.0.10-master.5
Daemon Ver:     1.1.3
Ti CLI Ver:     5.1.1
Alloy Ver:      1.13.7
Node Ver:       8.9.1
NPM Ver:        5.5.1
Java Ver:       10.0.2
Devices:        ⇨ google Nexus 5 (Android 6.0.1)
                ⇨ google Pixel (Android 9)

**Pull request merged**